dread.js

```diff
@@ -20,12 +20,15 @@ const JitterType = Object.freeze({
 });
 
 function exp(options = {}) {
-    const { base = 100, factor = 2, limit = 10000, jitter = JitterType.FULL} = options;
+	const { base = 100, factor = 2, limit = 10000} = options;
+	let { jitter = JitterType.FULL } = options;
 
     assert(Number.isInteger(base), 'base must be an integer');
     assert(Number.isInteger(factor), 'factor must be an integer');
 	assert(Number.isInteger(limit), 'limit must be an integer');
-	assert(typeof jitter !== 'boolean', `jitter type boolean is deprecated, use JitterType ${util.inspect(JitterType)}`);
+	if(typeof jitter === 'boolean') {
```

Comment:
I don't have eslint set up to enforce it, but space after if ( please

Reply:
ah I see a few spots of for( too, nevermind, i'll add linting and format things in a separate pr